pkg/workflows/wasm/v2/syncer_wasip1.go

```diff
@@ -0,0 +1,8 @@
+package wasm
```

Comment:
Why is this file named "syncer"?

Reply:
I copy/pasted from V1, but renamed and modified a lot. I missed renaming one file.